src/components/AnalysisContent/Actions/DownloadPDF.js

```diff
@@ -111,7 +112,8 @@ const createPdfStyles = StyleSheet =>
   });
 
 const createPdf = (Document, Image, Link, Page, Text, View) => {
-  function AnalysisPDF({ classes, topic, data, takwimu }) {
+  function AnalysisPDF({ topic, data, takwimu }) {
+    const classes = useStyles();
```

Comment:
This should remain `function AnalysisPDF({ classes, topic, data, takwimu })`... the `classes` actually means classes for `PDF` and not web styles.

Probably bad naming on my part... Change it to `pdfClasses` or `pdfStyles` or something better.